service/src/test/java/com/ge/predix/acs/testutils/Constants.java

```diff
@@ -0,0 +1,75 @@
+package com.ge.predix.acs.testutils;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.ge.predix.acs.model.Attribute;
+
+public class Constants {
```

Comment:
https://github.com/predix/acs/commit/8e578aa6d97ca901f651f5115016c00bff3bbba9  - renamed constants to improve understandibility.